RecyclerView/app/src/main/java/net/boondockradio/recyclerview/StatesDelegate.java

```diff
@@ -0,0 +1,6 @@
+package net.boondockradio.recyclerview;
+
+public interface StatesDelegate {
```

Comment:
In general, `Listener` or `Callback` use as an inner class.
http://qiita.com/ushi3_jp/items/ba0df2df6b0b77278f14